tests/MoBi.Tests/Core/CheckNameVisitorSpecs.cs

```diff
@@ -149,23 +150,23 @@ protected override void Context()
       {
          base.Context();
          _module = new Module();
-         _initialConditionsBuildingBlock = new InitialConditionsBuildingBlock { Name = _changedName };
+         _initialConditionsBuildingBlock = new InitialConditionsBuildingBlock {Name = _changedName};
          _initialCondition = new InitialCondition();
-         _path = new ObjectPath(new[] { "A", "B", _changedName });
+         _path = new ObjectPath(new[] {"A", "B", _changedName});
          _initialCondition.Path = _path;
          _initialConditionsBuildingBlock.Add(_initialCondition);
 
          _module.Add(_initialConditionsBuildingBlock);
 
          _initialCondition2 = new InitialCondition();
-         _path = new ObjectPath(new[] { "A", _changedName, "B" });
+         _path = new ObjectPath(new[] {"A", _changedName, "B"});
          _initialCondition2.Path = _path;
          _initialConditionsBuildingBlock.Add(_initialCondition2);
       }
 
       protected override void Because()
       {
-         _changes = sut.GetPossibleChangesFrom(_changedObject, _newName, _module, _changedName);
```

Comment:
funny how changing the actual code to be used as in producttion shows wrong usage in the tests. The fact that it worked here is only because of how the tree is being traversed (via visitor)